docs/_posts/2014-08-31-sor.markdown

```diff
@@ -125,32 +135,28 @@ Request Parameters:
+    mechanism for iterating related records other than a table.  In general, good practice is to create a table
     for every combination of data type and client, for example "review:testcustomer".  By convention, use colons `:`
     to separate fields in your table names.
-
-*   `audit` - required - An [O-Rison-encoded] (http://mjtemplate.org/examples/rison.html) map containing
+*   `audit` - required - An [O-Rison-encoded](https://github.com/Nanonid/rison) map containing
     information that can be used to trace changes to an object and debug applications that use EmoDB.
-    If your client is written in Java, you may use the [rison] (https://github.com/bazaarvoice/rison)
-    project to implement the O-Rison encoding.  For other languages, see http://mjtemplate.org/examples/rison.html.
-    There are a few [pre-defined keys] (https://github.com/bazaarvoice/emodb/blob/master/common/api/src/main/java/com/bazaarvoice/emodb/sor/api/Audit.java)
+    If your client is written in Java, you may use the [rison](https://github.com/bazaarvoice/rison)
+    project to implement the O-Rison encoding.  For other languages, see [here](https://github.com/Nanonid/rison).
+    There are a few [pre-defined keys](https://github.com/bazaarvoice/emodb/blob/master/common/api/src/main/java/com/bazaarvoice/emodb/sor/api/Audit.java)
     in Audit.java that clients are encouraged to use.  You may pass an empty map of audit information
     (encoded as an empty string), but this is discouraged.  After applying the O-Rison encoding, don't
     forget that, as with all url query parameters, the audit argument must be UTF-8 URI-encoded.  There
     are no intrinsic limits on the size of the audit map, but in practice it is limited by the maximum
     length of the URL.
-*   `options` - required - An [O-Rison-encoded] (http://mjtemplate.org/examples/rison.html) map containing options
+*   `options` - required - An [O-Rison-encoded](https://github.com/Nanonid/rison) map containing options
     that affect the internal storage of documents in the table.  For now, the only option is "placement" which
-    must be either "ugc_global:ugc" (for user generated data) or "catalog_global:cat" (for client metadata and product
-    catalog data).
+    must be one of the configured placements.
 
 Example `placement` Values :
 
-*   `ugc_global:ugc` - Use this placement for user generated data replicated globally.
-
-*   `catalog_global:cat` - Use this placement for client metadata and product catalog data.
-
-*   `app_global:default` - Use this placement for arbitrary data from generally low traffic applications.
+*   `ugc_global:ugc` - Sample placement for user generated data replicated globally.
+*   `catalog_global:cat` - Sample placement for client metadata and product catalog data.
+*   `app_global:default` - Sample placement for arbitrary data from generally low traffic applications.
```

Comment:
Not sure if you would want to do it as a part of this PR, but somewhere we need to note that the convention for placement is: <keyspace>:<prefix of delta table>